dts/bindings/gpio/adi,max14906-gpio.yaml

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2024 Analog Devices Inc.
+# Copyright (c) 2024 BayLibre SAS
+# SPDX-License-Identifier: Apache-2.0
+
+description: ADI MAX14906 quad industrial Input/Output with advanced diagnostics
+
+compatible: "adi,max14906-gpio"
+
+properties:
+  "#gpio-cells":
+    const: 2
+  ngpios:
+    type: int
+    required: true
+    const: 4
+    description: Number of gpios supported
+  drdy-gpios:
+    description: |
+      High-Side Open-Drain Output. READY is passive low when the internal
+      logic supply is higher than the UVLO threshold, indicating that the
+      registers have adequate supply voltage.
+    type: phandle-array
+  fault-gpios:
+    description: |
+      Fault pin indicates when there is Fault state in either FAULT1 or FAULT2
+      bothe of which are cleaned on read once problem is not persistent.
+    type: phandle-array
+  sync-gpios:
+    description: |
+      Latch the data so it could be read (partially duplicate CS).
+    type: phandle-array
+  en-gpios:
+    description: |
+      DOI Enable Pin. Drive the EN pin high to enable the DOI_ outputs.
+      Drive EN low to disable/three-state all DOI_ outputs.
+    type: phandle-array
+  crc-en:
+    description: |
+      Notify driver if crc pin is enabled.
+    type: boolean
+  spi-addr:
+    type: int
+    default: 0
+    required: true
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+    description: |
+      On MAX14906PMB module default address is 0 (A0-LOW, A1-LOW)
+      Selectable device address, configurable from A0 and A1
+  ow-en:
+    type: array
+    default: [0, 0, 0, 0]
+    description: |
+      Default values are from documentation.
+      Enable or disable open-wire functionality per channel.
+      - 0 mean enable
+      - 1 mean disable
+      channels indentation start from CH0...CH3
+  vdd-ov-en:
+    type: array
+    default: [0, 0, 0, 0]
+    description: |
+      Default values are from documentation.
+      VDDOVEN
+      Enable or disable open-wire functionality per channel.
+      - 0 mean enable
+      - 1 mean disable
```

Comment:
Is this correct? I think it may be inverted

Reply:
Yes it is inverted. Unfortunately multiplicative by copy/paste. Fixed on all locations.